service/src/main/java/bio/terra/cbas/dependencies/wds/WdsClientUtils.java

```diff
@@ -0,0 +1,26 @@
+package bio.terra.cbas.dependencies.wds;
+
+public class WdsClientUtils {
+
+  /**
+   * Given an ApiException message from the WDS client, attempt to parse a human-readable error out
+   * of the formatted message. ApiException.getMessage() returns a String in the format: "Message:
+   * %s%nHTTP response code: %s%nHTTP response body: %s%nHTTP response headers: %s" and we'd like to
+   * get just that first %s, representing the actual message.
+   *
+   * @param excMsg the WDS client exception message
+   * @return the human-readable string, or the identity if it could not be parsed
+   */
+  public static String extractErrorMessage(String excMsg) {
```

Comment:
It looks like this is always called (from non-test code) with `e.getMessage()`. You could instead make this accept `Exception` and do the `e.getMessage()` here.